lib/components/Stack/Stack.tsx

```diff
@@ -21,39 +26,49 @@ const alignToDisplay = {
   right: 'flex',
 } as const;
 
-export const useStackItem = ({ align, component, space }: UseStackProps) => {
-  const styles = useStyles(styleRefs);
-
-  return useBoxStyles({
-    component,
-    className: styles.excludingLast,
-    paddingBottom: space,
+export const useStackItem = ({ align, space }: UseStackItemProps) =>
+  ({
```

Comment:
Modified this function to return Box props rather than a `className` string. This is so that I can dynamically change the responsive `display` value to `none` for hidden items. The alternative was to create a `StackItem` component, which I did begin to attempt, but that seemed a lot heavier than this approach.